src/jv_dtoa.c

```diff
@@ -299,7 +299,7 @@ extern "C" {
 #endif
 
 #if defined(IEEE_8087) + defined(IEEE_MC68k) + defined(VAX) + defined(IBM) != 1
-Exactly one of IEEE_8087, IEEE_MC68k, VAX, or IBM should be defined.
+#  error Exactly one of IEEE_8087, IEEE_MC68k, VAX, or IBM should be defined.
```

Comment:
This is a vendored file, but I guess we change it.
This would be the only directive in the file that uses indentation though

Reply:
I'm not sure what you mean with vendored, can you elaborate? In any case, the indentation can be removed, it's something I do without thinking about it.

Reply:
I think he menas that `jv_dota.c` is a copy of the half-famous "dtoa.c" code

Reply:
Nevermind, I saw you merged this already. :)

Reply:
It is a library file that is included in the repository, but it is not written by us